### PR TITLE
Replace Range Search dialog with a modal

### PR DESCRIPTION
**Pull request by shucontech**
Signed-off-by: Saksham Gupta <shucontech@gmail.com>

### Description

Please describe your pull request.

Fixes #

Before submitting pull request, please review the following checklist:

- [ ] Make sure you have read our [CONTRIBUTING.md](https://github.com/phpmyadmin/phpmyadmin/blob/master/CONTRIBUTING.md) document.
- [ ] Make sure you are making a pull request against the correct branch. For example, for bug fixes in a released version use the corresponding QA branch and for new features use the _master_ branch. If you have a doubt, you can ask as a comment in the bug report or on the mailing list.
- [ ] Every commit has proper `Signed-off-by` line as described in our [DCO](https://github.com/phpmyadmin/phpmyadmin/blob/master/DCO). This ensures that the work you're submitting is your own creation.
- [ ] Every commit has a descriptive commit message.
- [ ] Every commit is needed on its own, if you have just minor fixes to previous commits, you can squash them.
- [ ] Any new functionality is covered by tests.

### CODE REVIEW

File: js/src/table/select.js [hunk 1]
```diff
@@ -319,8 +319,17 @@ AJAX.registerOnload('table/select.js', function () {
                             ? '(' + Messages.strColumnMax +
                                 ' ' + response.column_data.max + ')'
                             : '';
-                        var buttonOptions = {};
-                        buttonOptions[Messages.strGo] = function () {
+                        $('#rangeSearchModal').modal('show');
+                        $('#rangeSearchLegend')[0].innerHTML = operator;
+                        $('#rangeSearchMin')[0].innerHTML = min;
+                        $('#rangeSearchMax')[0].innerHTML = max;
```
Comment (williamdes) [inline js/src/table/select.js:325]: I am not sure this is a very safe way

File: js/src/table/select.js [hunk 2]
```diff
@@ -319,8 +319,17 @@ AJAX.registerOnload('table/select.js', function () {
                             ? '(' + Messages.strColumnMax +
                                 ' ' + response.column_data.max + ')'
                             : '';
-                        var buttonOptions = {};
-                        buttonOptions[Messages.strGo] = function () {
+                        $('#rangeSearchModal').modal('show');
+                        $('#rangeSearchLegend').first().html(operator);
+                        $('#rangeSearchMin').first().html(min);
+                        $('#rangeSearchMax').first().html(max);
```
Comment (williamdes) [inline js/src/table/select.js:325]: this is pure text, right ?
.text()  would be safer in that case

File: js/src/table/select.js [hunk 3]
```diff
@@ -319,8 +319,17 @@ AJAX.registerOnload('table/select.js', function () {
...
+                        // Reset input values on reuse
+                        $('#min_value')[0].value = '';
+                        $('#max_value')[0].value = '';
```
Comment (williamdes) [inline js/src/table/select.js:328]: .val() does not work ?

File: js/src/table/select.js (hunk 2 above)
Comment (shucontech) [reply js/src/table/select.js:325]: Ahh yes, thanks

File: js/src/table/select.js (hunk 3 above)
Comment (shucontech) [reply js/src/table/select.js:328]: Oh my bad, I was trying `.value()`

**Review by MauricioFauth — APPROVED** [review]

**Pull request merged**